robottelo/cli/hammer.py

```diff
@@ -157,7 +157,8 @@ def parse_info(output):
             # entity name like 'test::params::keys'
             if line.find(':') != -1 and not line.find('::') != -1:
                 key, value = line.lstrip().split(":", 1)
-            elif line.find('=>') != -1:
+            elif line.find('=>') != -1 and len(
+                    line.lstrip().split(" =>", 1)) == 2:
```

Comment:
Should be pretty harmless change. It basically means set/unpack values only if there's a pair to unpack. For nested locations there can be following output:
```
Hostgroups:         

Parameters:         
     =>
Created at:         2017/09/11 14:42:57
```
when the location has parent location set (so it inherits its parameters), but there's nothing to inherit. Such case causes failures as of now, since line `     =>` don't have a valid pair of key & value.

Reply:
why not use csv and json?

Reply:
@abalakh got it , you had an exception when parsing this line

Reply:
Any impact on other test modules?

Reply:
Nice 
And it will skip without any message? Maybe we can put something into log ?

Reply:
Log already contains full command output.
Just to make sure we're on the same page - we're talking about parameters and other key-value pairs, separated by `=>` here.
Example above shows it:
```
Hostgroups:         

Parameters:         
     =>
Created at:         2017/09/11 14:42:57
```
Here parameters should be empty since nothing is really there. Putting log message stating that parameters are empty? Hm, why not putting similar message about empty hostgroups then? If the test is failing because expected item is missing, then test log contains full hammer output and it's easy to compare error and actual output. Or what's your idea, maybe i didn't get it?